pages/portfolio.html

```diff
@@ -4,9 +4,75 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Abby's Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <link href="/styles/portfolio.css" rel="stylesheet">
 </head>
+
 <body>
-  
+<div class="container">
+<header>
+  <h1>Abby Choi</h1>
+  <nav>
+    <ul>
+    <li><a href="/pages/index.html">Home</a></li>
+    <li><a href="/pages/about.html">About Me</a></li>
+    <li><a id="portfolio" href="/pages/portfolio.html">Portfolio</a></li>
+    </ul>
+  </nav>
+</header>
```

Comment:
As you might've experienced, this navbar and footer HTML code is repetitive, which is unavoidable with HTML. But this can be component-ized in React and imported across files. :)